applications/ShapeOptimizationApplication/python_scripts/custom_ios/wrl_io.py

```diff
@@ -53,14 +55,28 @@ def ReadModelPart(self, model_part):
                 node_id = i + nodes_shift
                 new_node = model_part.CreateNewNode(node_id, *node)
                 sub_model_part.AddNode(new_node, 0)
+                node_usage_count[node_id] = 0
 
             for i, triangle in enumerate(shape.triangles):
                 triangle_id = i + triangles_shift
                 node_ids = [x + nodes_shift for x in triangle]
 
+                for node_id in node_ids:
+                    node_usage_count[node_id] += 1
+
                 new_condition = model_part.CreateNewCondition("SurfaceCondition3D3N", triangle_id, node_ids, new_property)
                 sub_model_part.AddCondition(new_condition)
 
             nodes_shift += len(shape.nodes)
             triangles_shift += len(shape.triangles)
+
+        counter = 0
```

Comment:
```suggestion
        # Remove flying nodes
        counter = 0
```

I think here a comment makes sense.